app/Http/Controllers/Api/Project/MediaController.php

```
@@ -89,6 +89,7 @@ public function getMedia()
      */
     public function getMediaLocal(array $params)
     {
+        $photoPlaceholderFilename = config('epicollect.media.photo_placeholder.filename');
```

PHPMD (2.15.0)

Avoid excessively long variable names like $photoPlaceholderFilename. Keep variable name length under 20. (Naming Rules)

(LongVariable)